articles/machine-learning/how-to-use-environments.md

```diff
@@ -415,6 +434,34 @@ Download a registered environment by using the following command.
 az ml environment download -n myenv -d downloaddir
 ```
 
+## Troubleshooting
```

Comment:
Typically, we don't include troubleshooting content like this unless it's a frequent and consistent ask from customers. Otherwise, we publish this type of information on our known issues page [here](https://docs.microsoft.com/en-us/azure/machine-learning/resource-known-issues#set-up-your-environment). Would this content better reside on the known issues page? If we move it, then I can include a link in the next steps to the troubleshooting content so users can go there for additional info. Thoughts?

Reply:
Sure, the known issues page should work for that content.

Reply:
Great. I'll spin up another PR to add this content to the Known issue page. Can you remove your troubleshooting section from your PR please?